page-objects/pages/NavigationTopPage.js

```diff
@@ -10,11 +10,32 @@ module.exports = {
         initialsDDLogout: 'a[data-testid="logout"]',
         nameBox: '.dropdown-name',
         userIcon: '.btn-avatar > a',
+        navItemHelp: 'li.nav-item.help-drodown',
+        initialsHelpButton: '.list-group-item.list-group-item-action:first-child',
+        initialsRequestOrProblemButton: '.list-group-item.list-group-item-action:nth-child(2) .link',
```

Comment:
please set the data-testids for all these selectors

Reply:
in client, in extra branch with the same name as in integration tests

Reply:
I didn't find any data-testids on branch SC-4994-/-add-testids-for-integration-tests-2 run locally.

Reply:
I set some data-testids